.ipynb_checkpoints/Untitled-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
+ "metadata": {},
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
@Parvfect re: this - IMHO it is best to name files individually when committing. 

i.e. do

```bash
git commit file1.py file2.py file3.py file4.py file5.py -m"my commit description"
```

rather than 

```bash
git commit * -m"my commit description"
``` 

because the latter, even though syntactically simpler, will frequently lead to unintended files being committed, even when one is not being particularly sloppy.